files/en-us/web/api/displaymediastreamconstraints/video/index.html

```diff
@@ -21,19 +21,15 @@
 ---
 <p>{{APIRef("Screen Capture API")}}</p>
 
-<p><span class="seoSummary">The {{domxref("DisplayMediaStreamConstraints")}} dictionary's
-    <strong><code>video</code></strong> property is used to configure the video track in
-    the stream returned by {{domxref("MediaDevices.getDisplayMedia",
-    "getDisplayMedia()")}}.</span> This value may be a Boolean, where <code>true</code>
-  specifies that a default selection of input source be made (typically the entire display
+<p>The {{domxref("DisplayMediaStreamConstraints")}} dictionary's <strong><code>video</code></strong> property is used to configure the video track in
+    the stream returned by {{domxref("MediaDevices.getDisplayMedia", "getDisplayMedia()")}}.</p>
+    
+<p>This value may be a Boolean, where <code>true</code> specifies that a default selection of input source be made (typically the entire display
```

Comment:
Like in the specs, we make the difference between the object Boolean and the boolean values (`true` and `false`) that are strings that convert to a boolean litteral. (without capitalization in the spec and in MDN). We are not yet consistent all over the place, but I recently replaced all the incorrect `{{jsxref("Boolean")}}` in Web/API.
```suggestion
<p>This value may be a boolean value, where <code>true</code> specifies that a default selection of input source be made (typically the entire display
```